main.py

```diff
@@ -40,6 +41,10 @@
         ]
     except KeyError:
         raise Exception("Check configuration: num_temp_measurements must be specified")
+if not CONFIG["simulation_hyperparameters"].get("limited_time_sampling", False):
```

Comment:
`LIMITED_TIME_SAMPLING = CONFIG["simulation_hyperparameters"].get("limited_time_sampling", False)` is equivalent (i think) and simpler.

Reply:
Implemented